src/vectorizationbase_compat/contract_pass.jl

```diff
@@ -107,14 +107,14 @@ function recursive_muladd_search!(call, argv, cnmul::Bool = false, csub::Bool =
                         csub = false
                     end
                     return true, cnmul, csub
-                end                
+                end
             end
         end
     end
     length(call.args) == 4, cnmul, csub
 end
-                          
-function capture_muladd(ex::Expr, mod, LHS = nothing)
+
+function capture_muladd(ex::Expr, mod, @nospecialize(LHS) = nothing)
```

Comment:
Same question as above: is there any risk that adding this `@nospecialize` will cause the runtime performance to be slower?

Reply:
In this case I don't think the argument is even used.

Reply:
It was once upon a time, but I changed where the `vfmadd` -> `vfmadd231` substitution takes place (it now happens during lowering, in `lower_compute!`).

It also no longer happens on Julia 1.6+. The reason for using `vfmadd231` was that LLVM used to not optimize many muladds well, choosing the wrong `vfmadd` version (overwriting the wrong registers), requiring the addition of move statements to copy register contents. So I used `asm call` to pick the correct version and avoid all the `vmovapd`s.
This has been fixed as of LLVM 11, which ships with Julia 1.6.

The commented out use of `LHS` is never getting un-commented, is what I'm trying to say.

Mind removing the argument altogether?